### ISSUE TITLE
[mdspan.accessor.reqmts] Name type `A` to `ACC`?

### ISSUE THREAD

**Issue by hewillk** [cplusplus/draft#6622]
The use of `A` in [[mdspan.accessor.reqmts]](https://eel.is/c++draft/mdspan.accessor.reqmts) refers to types that meet accessor policy requirements, which makes it similar to "A" of "A type" and seems to reduce the comfort of reading.
Is it worth renaming `A` to `ACC` or is it just wishful thinking on my part?

**Comment by eisenwave** [comment]
I agree that `A` isn't an ideal choice, although `ACC` is a bit lengthy and unconventional.

Maybe just use `T`?

**Comment by frederick-vs-ja** [comment]
`ACC` sounds like accumulator to me. I think we should use plain `T`.
